docs/xcm/integration/add-to-portal.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 5
+---
+
+# Add assets into Astar Portal
+
+Once a channel is opened between parachains and assets have been registered, developers can create a PR to [our portal](https://github.com/AstarNetwork/astar-apps) for adding assets to our portal. Here’s what you need to know about the integration of the XCM assets into our portal.
+
+## Define the parachains information
+
+1. Add the `Chain name` at the [Chain](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
```

Comment:
This is not a documentation comment it is an idea for a small code refactoring. Lets move point 1, 2 and 3 to parachain repository implementation. In that case we will have a single class where changes needs to be implemented. What do you think?

Reply:
Cool! Could you create a PR because I'm not available today.

Reply:
Sure. I'll do it later today or tomorrow